mk/cfg/asmjs-unknown-emscripten.mk

```diff
@@ -0,0 +1,27 @@
+# asmjs-unknown-emscripten configuration
+CC_asmjs-unknown-emscripten=emcc
+CXX_asmjs-unknown-emscripten=em++
+CPP_asmjs-unknown-emscripten=$(CPP)
+AR_asmjs-unknown-emscripten=emar
+CFG_LIB_NAME_asmjs-unknown-emscripten=lib$(1).so
+CFG_STATIC_LIB_NAME_asmjs-unknown-emscripten=lib$(1).a
+CFG_LIB_GLOB_asmjs-unknown-emscripten=lib$(1)-*.so
+CFG_LIB_DSYM_GLOB_asmjs-unknown-emscripten=lib$(1)-*.dylib.dSYM
+CFG_JEMALLOC_CFLAGS_asmjs-unknown-emscripten := -m32 $(CFLAGS)
+# NB: The EMSCRIPTEN environment variable is set by the emscripten SDK.
+# This is only needed here to so the compiler-rt build can find unwind.h,
+# but the asmjs target *doesn't even link to compiler-rt*.
```

Comment:
Oh wait it looks like `no_compiler_rt` is already set, I wonder if we could avoid building compiler-rt like libbacktrace is avoided?